.gitignore

```diff
@@ -24,5 +24,5 @@ tilt_config.json
 localdev/terraform/gitlab/project.url
 *.tgz
 *.DS_Store
-kubechecks
+/kubechecks
```

Comment:
I think this was accidentally added.

Reply:
This was intentional - the only `kubechecks` file that should be ignored is when it's built and dumped at the root of the repo. Any other `kubechecks` files, as far as we currently know, should be committed.